lib/seed_helper.rb

```diff
@@ -169,7 +169,7 @@ def test_template_requirements(template_name, requirements)
         if plugin.nil?
           logger.info("Template #{template_name} requires plugin #{r['plugin']}, skipping import.")
           return false
-        elsif r['version'] && (Gem::Version.new(plugin.version) < Gem::Version.new(r['version']))
+        elsif r['version'] && (Gem::Version.new(plugin.version).release < Gem::Version.new(r['version']))
```

Comment:
Do we care about specifying a perrelease version in the template? because if we do this might lead to unexpected result:
```ruby
[11] pry(main)> Gem::Version.new('1.2.0.rc1').release < Gem::Version.new('1.2.0.rc2')
=> false

Reply:
I don't think we do. I don't see a template to require a rc1.

Reply:
no, let's keep it simple and only keep versioning on GA versions, technically we may want to depend on something we added only in rc, but in real world, we care about Katello 3.14, not any of it's RC.